### PR TITLE
Cristian

### PR DESCRIPTION
**Pull request by Cristian01021**

### CODE REVIEW

File: list.html [hunk 1]
```diff
@@ -19,17 +19,6 @@
...
-    <section class="nav">
-        <a class="nav-link" id="movie-link" href="#movie-section">
-            <i class="large material-icons">movie</i>
-            <p>My Movies</p>
-        </a>
-    
-        <a class="nav-link" id="tv-link" href="#tv-section">
-            <i class="large material-icons">tv</i>
-            <p>My TV</p>
-        </a>
```
Comment (notalisk) [inline list.html:32]: Why did we remove those?

File: index.html [hunk 2]
```diff
@@ -48,41 +48,48 @@ <h1>Welcome to Reelgood</h1>
...
         <button class="store-want-btn">Want to see</button>
     </div>
 
     <!-- Card 5 -->
     <div class="card" style="display:none">
-        <img src="" id="card5" alt="Card 5">
+        <img src="" id="card5" class="card-img"  alt="Card 5">
         <h3 class="movie" id="title5"></h3>
+        
         <button class="store-loved-btn">Seen it (Love)</button>
```
Comment (notalisk) [inline index.html:93]: why does this last card not have a rating input?

File: assets/js/script-index.js [hunk 3]
```diff
@@ -136,49 +136,53 @@ movieElements.forEach(function(movieElement) {
...
-  
+  var rating = $(this).siblings("input").val();
   var seenMovie = {
     title: title,
-    like: true
+    like: true,
+    rating: rating, 
   }
 
-  var seenMovies = JSON.parse(localStorage.getItem("seenMovies"));
```
Comment (notalisk) [inline assets/js/script-index.js:145]: this should pull from seenMovies, not lovedMovies

Comment (notalisk) [reply assets/js/script-index.js:145]: Same with all the other things you changed here

File: list.html [hunk 4]
```diff
@@ -86,8 +75,6 @@ <h2>Want To See It</h2>
                 <thead>
                     <tr>
                         <th id="movieH">Movie</th>
-                        <th id="loveH">Reel-y Love it</th>
-                        <th id="hateH">Reel-y Hate it</th>
```
Comment (notalisk) [inline list.html:90]: Also why did we remove those

**Review by notalisk — CHANGES_REQUESTED** [review]
nice job with the css. I am a bit unsure of the changes you made in the js though, I think maybe you missed the names the local storage is using?

File: list.html (hunk 1 above)
Comment (Cristian01021) [reply list.html:32]: i thought we werent having a tv section since were only doing movies

File: index.html (hunk 2 above)
Comment (Cristian01021) [reply index.html:93]: i think i just forgot to add them since i was trying to think of what i could do

**Pull request closed without merging**